src/controllers/eventController.js

```diff
@@ -0,0 +1,27 @@
+const { eventService } = require('../services');
+const { throwError } = require('../utils');
+const { v4: uuidv4 } = require('uuid');
+
+const userByLikeController = async (req, res) => {
+  const userId = req.userData;
+  const reactionType = req.query.reaction_type || 'exited';
```

Comment:
```suggestion
  const reactionType = req.query.reaction-type || 'exited';
```
query parameter를 포함한 url & uri에서는 언더바(`_`) 보단는 하이픈(`-`)으로 단어를 구분하는 것이 보다 일반적인 경우입니다.